compiler/rustc_passes/src/dead.rs

```diff
@@ -164,10 +164,9 @@ impl<'tcx> MarkSymbolVisitor<'tcx> {
                     if let (Res::Local(id_l), Res::Local(id_r)) = (
                         typeck_results.qpath_res(qpath_l, lhs.hir_id),
                         typeck_results.qpath_res(qpath_r, rhs.hir_id),
-                    ) {
-                        if id_l == id_r {
-                            return true;
-                        }
+                    ) && id_l == id_r
```

Comment:
Side-note: difficult to tell what is meaningful code change with all of these tiny codemods everywhere in this file :/

Reply:
I forgot reverting this change.